### DISCUSSION TITLE
Error while fetching blocks from private blockchain (POW Based)

### DISCUSSION THREAD

**Discussion by malkeetnetsmartz** [blockscout/blockscout#9472] [Support]
I am getting this while  blockscout app fetching blocks from the private blockchain.

How can i pass auth token to blokscout or any other flag to overcome this issue because it causes the blockchain explorer to stuck in indexing blockchain

![image](https://github.com/blockscout/blockscout/assets/153892101/03c1d330-caf0-475d-9e76-a2954a569c5e)

 

application=ethereum_jsonrpc fetcher=block_realtime [error] Failed to decode Ethereum JSONRPC response:

  request:

    url: http://geth.local:8741

    body: [{"id":0,"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["latest",false]}]

  response:

    status code: 401

    body: token contains an invalid number of segments
    
    
    
    
![image](https://github.com/blockscout/blockscout/assets/153892101/7f8cb87c-810d-49ca-9aab-36761b85f4ef)

**Comment by ekawibisono** [comment]
Hello

What command you run geth pow ?

**Reply by malkeetnetsmartz** [reply]
geth --datadir mynode --allow-insecure-unlock --unlock 0 --password mynode/secret.txt --networkid 209445 --http --http.addr 0.0.0.0 --http.port 8765 --http.api eth,net,web3,personal,miner --ws --ws.addr 0.0.0.0 --ws.port 8978 --ws.origins=*  --authrpc.addr 0.0.0.0 --authrpc.port 8741 --authrpc.jwtsecret mynode/jwt.hex

**Reply by ekawibisono** [reply]
try change with API This
--http.api eth,txpool,debug,web3,net,personal,admin,miner

**Reply by malkeetnetsmartz** [reply]
![image](https://github.com/blockscout/blockscout/assets/153892101/b4c06d68-0696-4999-a581-1767b86441e9)


Geth command used is : 
 geth --datadir mynode --allow-insecure-unlock --unlock 0 --password mynode/secret.txt --networkid 22334455 --http --http.addr 0.0.0.0 --http.port 8765 --http.api eth,txpool,debug,web3,net,personal,admin,miner --ws --ws.addr 0.0.0.0 --ws.port 8978 --ws.api eth,net,web3 --authrpc.addr 0.0.0.0 --authrpc.port 8741 --authrpc.jwtsecret mynode/jwt.hex --remotedb postgresql://admin:adminpassword@10.1.2.46:5432/blockscout --ws.origins=* console
 
 Deploying blockscout manually
  
 Environment Variables : 
 
export ETHEREUM_JSONRPC_VARIANT=geth 
export MIX_ENV=prod 
export ETHEREUM_JSONRPC_HTTP_URL=http://geth.local:8741 
export ETHEREUM_JSONRPC_TRACE_URL=http://geth.local:8765 
export ETHEREUM_JSONRPC_WS_URL=ws://geth.local:8978 
export JSON_RPC=http://geth.local:8765 &&
export DATABASE_URL=postgresql://admin:adminpassword@192.1.126:5432/blockscout 
export API_V2_ENABLED=true 
export PORT=3001  
export COIN=  
export COIN_NAME= CHEYTRR
export DISPLAY_TOKEN_ICONS=true 
export INDEXER_DISABLE_INTERNAL_TRANSACTIONS_FETCHER=true 
export INDEXER_DISABLE_PENDING_TRANSACTIONS_FETCHER=true 
export ECTO_USE_SSL=false 
export CHAIN_ID=889798 
export ETHEREUM_JSONRPC_HTTP_INSECURE=true 
export ENABLE_EXPERIMENTAL_FEATURES=true

**Reply by GimluCom** [reply]
Why are you trying to use `authrpc.port` instead of `http.port`? Please change it to this:
```sh
export ETHEREUM_JSONRPC_HTTP_URL=http://geth.local:8765/
export ETHEREUM_JSONRPC_TRACE_URL=http://geth.local:8765/
```
